EMACS.md

```diff
@@ -0,0 +1,24 @@
+# Running import-js in Emacs
+
+1. Install import-js
+  * `gem install import-js`
+2. Configure import-js
+  * See [Configuration](README.md#configuration)
+2. Install import-js.el for Emacs
+  * This is not yet available on elpa/melpa, so for now this means dropping
+    import-js.el somewhere in your .emacs.d
```

Comment:
I'm assuming you mean that the file should be grabbed from within the gem? Or did you mean grab it from the repo directly?